packages/eslint-config-base/index.js

```diff
@@ -107,7 +107,7 @@ module.exports =  {
         ],
       },
     }],
-    'complexity': ['warn', 10],
+    'complexity': ['warn', 15],
```

Comment:
Don't you think 10 is still okay for a warning? Where was it mentioned that we should be increasing this to 15? Best practices mention the value should be around 7-8, so we are already above that with this.